Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -2,18 +2,34 @@ Enable Hubble for Cluster-Wide Visibility
 =========================================
 
 Hubble is the component for observability in Cilium. To obtain cluster-wide
-visibility into your network traffic, deploy Hubble Relay and the UI with the
-following Helm upgrade command on your existing installation
-(Cilium agent pods will be restarted in the process).
-
-.. parsed-literal::
-
-   helm upgrade cilium |CHART_RELEASE| \\
-      --namespace $CILIUM_NAMESPACE \\
-      --reuse-values \\
-      --set hubble.listenAddress=":4244" \\
-      --set hubble.relay.enabled=true \\
-      --set hubble.ui.enabled=true
+visibility into your network traffic, deploy Hubble Relay and the UI as follows
+on your existing installation:
+
+.. tabs::
+
+    .. group-tab:: Installation via Helm
+
+        If you installed Cilium via ``helm install``, you may enable Hubble
+        Relay and UI with the following command:
+
+        .. parsed-literal::
+
+           helm upgrade cilium |CHART_RELEASE| \\
+              --namespace $CILIUM_NAMESPACE \\
+              --reuse-values \\
+              --set hubble.listenAddress=":4244" \\
```

Comment:
do you still need this even if it's specified in values.yaml?

Reply:
Hm. Yes, on `master`, we should probably remove this. I left it in because if we decide to backport this to 1.9, we probably want to keep it for users who are using the Cilium 1.9.0 chart. But I'll remove it for the master version.

Reply:
Removed https://github.com/cilium/cilium/compare/75f993ed3fe980e7098a6b7041f8d6530d9ab083..5f00eeeef22b8d4d6b024f53b58791e229307750